companies/junyanwang/A Frozen SingAlong Celebration.php

```diff
@@ -87,6 +87,11 @@
          </div> <!-- /entry -->	      
 
       </div> <!-- /portfolio-content -->
+
+    <?php
+    $include_name = basename(__FILE__, '.php');
+    include "$path/comment&rating.php"
```

Comment:
pass in the $include_name to a function inside php. NO ONE knows where when and why you use it.